corehq/apps/reports/filters/forms.py

```diff
@@ -45,8 +45,7 @@ def display_lang(self):
         """
             This should return the lang code of the language being used to view this form.
         """
-        # todo make this functional
-        return 'en'
+        return self.request.couch_user.language or 'en'
```

Comment:
FormsByApplicationFilter extends BaseReportFilter, and `request` is a mandatory parameter, so we know it will always be set. I'm pretty sure all reports are only available to authenticated users, but Just In Case, it's probably wise to check that couch_user is set. Thanks @czue.